src/gui/Src/Gui/DataCopyDialog.cpp

```diff
@@ -264,7 +278,12 @@ void DataCopyDialog::printData(DataType type)
         for(int i = 0; i < numIPs; i++)
         {
             if(i)
-                data += ", ";
+            {
+                if((i % mTypes[mIndex].itemsPerLine) == 0)

```

Comment:
I don't really see it being an issue right now, but this and the other two instances of it have the potential  to divide by zero in the future if itemsPerLine is ever set to zero (or someone blindly copy-pastes this code elsewhere)

Reply:
Good point!